hack/local-setup.sh

```diff
@@ -135,22 +135,34 @@ deployDashboard() {
 
   kubectl proxy --context kind-${clusterName} --port ${port} &
   proxyPID=$!
+  echo $proxyPID >> /tmp/dashboard_pids
 
   echo -ne "\n\n\tAccess Kubernetes Dashboard\n\n"
   echo -ne "\t\t\t* The dashboard is available at http://localhost:$port/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/\n"
-  echo -ne "\t\t\t* To kill the proxy background process, run:\n"
-  echo -ne "\t\t\t\tkill $proxyPID\n"
   echo -ne "\t\tAccess the dashboard using the following Bearer Token: $token\n"
 }
 
+stopProxies() {
+  if [[ -f /tmp/dashboard_pids ]]; then
+    echo "Stopping existing proxies"
+    while read p; do
+      kill $p
+    done </tmp/dashboard_pids
+    rm /tmp/dashboard_pids
+  fi
+}
+
+# Stop proxies to the existing clusters
+stopProxies
+
 #Delete existing kind clusters
 clusterCount=$(${KIND_BIN} get clusters | grep ${KIND_CLUSTER_PREFIX} | wc -l)
 if ! [[ $clusterCount =~ "0" ]] ; then
   echo "Deleting previous kind clusters."
   ${KIND_BIN} get clusters | grep ${KIND_CLUSTER_PREFIX} | xargs ${KIND_BIN} delete clusters
 fi
```

Comment:
Would a separate script make sense `local-cleanup.sh` that can be called at the start of this script, and also at any time to tear down resources/processes created by local-setup? It could delete the clusters as well as stop any dashboard proxies it started.

Reply:
I like that idea. I have created an initial version of that in another PR #67. I'll add the step of killing the proxy processes on top of it